runtime/binding-grpc/src/main/java/io/aklivity/zilla/runtime/binding/grpc/internal/config/GrpcBindingConfig.java

```diff
@@ -95,10 +94,14 @@ public GrpcBindingConfig(
         this.schemaIds = new Int2IntHashMap(-1);
         this.handlersById = new Long2ObjectHashMap<>();
         this.protobufsBySchemaId = new Int2ObjectHashMap<>();
-        this.catalogs = binding.catalogs != null ? binding.catalogs : new ObjectHashSet<>();
+        this.catalogs = binding.catalogs != null ? binding.catalogs : List.of();
```

Comment:
Perhaps have a default empty list of catalogs in the `BindingConfig` parser, so we don't need the null check here.